### ISSUE TITLE
9FUS YI 1080p Home variant support?

### ISSUE THREAD

**Issue by punkonjunk** [TheCrypt0/yi-hack-v4#201]
Is this model in the near future for support?

**Comment by pejotigrek** [comment]
+1 from me. just got the camera & looking for firmware. is there any roadmap on this?

**Comment by benogen** [comment]
This should work for you, right: https://github.com/roleoroleo/yi-hack-MStar ?

**Comment by pejotigrek** [comment]
...aaand it works great! thanks for the tip @benogen

**Comment by nstrelow** [comment]
Is 9FUS using the MStar platform and needs https://github.com/roleoroleo/yi-hack-MStar ?

**Comment by andyb2** [comment]
> Is 9FUS using the MStar platform and needs https://github.com/roleoroleo/yi-hack-MStar ?

There is a second version of 9FUS that is not MSTAR and it uses firmware 8.2.0.0A . There is no fix for it yet but it is being worked on by  roleoroleo. Could be a while. If your firmware is 4.5.0.0... then its probably MSTAR

**Comment by pbanj** [comment]
https://github.com/roleoroleo/yi-hack-Allwinner just flashed this on mine, seems to be working well

**Comment by njbendall** [comment]
> https://github.com/roleoroleo/yi-hack-Allwinner just flashed this on mine, seems to be working well

How long did it take to flash Allwinner because mine has been running for 45mins now with a flashing blue light?

**Comment by scruff-01** [comment]
What firmware is on the cam?

**Comment by njbendall** [comment]
> What firmware is on the cam?

8.2.0.0A_202007211802 currently

**Comment by scruff-01** [comment]
Thats correct for allwinner. Make sure to format a low capacity sd card to fat 32. The sd cards have been temper mental for allwinner

**Comment by njbendall** [comment]
right, okay. I have another SD card on its way tomorrow anyway so I'll see how that goes. I took out the other card and rebooted the camera and it was fine. just weird I got stuck there...

**Comment by scruff-01** [comment]
Yi cams are finicky with sd cards

**Comment by njbendall** [comment]
I might open a separate "issue" for this but I have just noticed that the app downloads firmware in an accessible format on Android phones, could it be swapped out with modified software then pushed to the device?

**Comment by scruff-01** [comment]
You would have to ask on the allwinner github but I do not think so
